packages/extension-ui/src/Popup/CreateAccount/Mnemonic.tsx

```diff
@@ -12,26 +12,13 @@ interface Props {
   seed: string;
 }
 
-const onCopy = (): void => {
-  const mnemonicSeedTextElement = document.querySelector('textarea');
-
-  if (!mnemonicSeedTextElement) {
-    return;
-  }
-
-  mnemonicSeedTextElement.select();
-
-  // eslint-disable-next-line deprecation/deprecation
-  document.execCommand('copy');
-};
-
 function Mnemonic ({ onNextStep, seed }: Props): React.ReactElement<Props> {
   const { t } = useTranslation();
   const [isMnemonicSaved, setIsMnemonicSaved] = useState(false);
   const { show } = useToast();
 
   const _onCopy = useCallback((): void => {
-    onCopy();
+    setIsMnemonicSaved(true);
```

Comment:
copying doesn't mean that the users have saved the mnemonic safely. I'd suggest to keep the manual checkbox as it was.

```suggestion
```

Reply:
This is still here BTW. Please remove it.